test/unit/date/format.js

```diff
@@ -443,6 +445,20 @@ QUnit.test( "should format various milliseconds (A+)", function( assert ) {
 /**
  *  Zone
  */
+QUnit.test( "should format timezone (z)", function( assert ) {
+	var date = new Date();
+	assert.dateFormat(date, "z", cldr, "PST", "America/Los_Angeles");
+	assert.dateFormat(date, "zz", cldr, "PST", "America/Los_Angeles");
+	assert.dateFormat(date, "zzz", cldr, "PST", "America/Los_Angeles");
+	assert.dateFormat(date, "zzzz", cldr, "Pacific Standard Time", "America/Los_Angeles");
+
+	//fall through to 'O' format
+	date = new FakeDate( 0 );
+	assert.dateFormat(date, "z", cldr, "GMT", "Etc/GMT");
```

Comment:
Nice! Though, I'm wondering... Should we use an invalid stuff that will never be supported like `"Foo/Bar"`?